### ISSUE TITLE
RecordXWrapper in hiora.interruptibility

### ISSUE THREAD

**Issue by Ac2zoom** [rmoehn/cartpole#1]
Encountering an import error in the ProcessedQNISQ notebook on line 6: `from gym.envs.safety.cartpole.record_x_wrapper import RecordXWrapper`

There isn't any cartpole module within safety, but there also isn't a record_x_wrapper class or module in any of the current modules located there.  Is this a custom branch/fork of gym, or am I missing something?

**Comment by rmoehn** [comment]
Yes, you concluded right. Sorry for confusing you! (I didn't expect that people would try and run the code.) Here you can find the `RecordXWrapper` in my fork of the Gym: https://github.com/rmoehn/gym/tree/interr-measure/gym/envs/safety/cartpole

Feel free to reach out again if you encounter more problems or are interested in anything.
